### PR TITLE
chore: update versions and fix code review errors

### PR DESCRIPTION
**Pull request by girazoki**
Fix some errors from the core review in #14.

```
·-----------------------------------------------·--------------·-------·
|                Error Type                     |  Occurences  | FIXED |
·······································································|
| Compiler version not fixed                    |           6  |     0 |
·······································································|
| Pure-functions should not read/change state   |           3  |     0 |
·······································································|
| Prefer external to public visibility level    |           2  |     0 |
·······································································|
| Use of assembly                               |           3  |     0 |
·······································································|
| Implicit visibility level                     |           6  |     6 |
·-----------------------------------------------·--------------·-------·
| TOTAL                                        |           20  |     8 |
·-----------------------------------------------·--------------·-------·
```

### Compiler version: NOT FIXED

This issue arises from the fact that minor differences in the compiler versions are still allowed. This is considered an error since future compiler versions may handle certain language constructions in a way the developer did not foresee.

### Pure-functions should not read/change state: NOT FIXED

This implies that the tool detected pure functions that potentially write the state of the contract.
These will not be fixed as the tool can not determine what assembly code does. Our assembly code only utilizes the mload instruction, so it does not modify any state of the contract. In fact the library does not have modifiable state.

### Prefer external to public visibility level: NOT FIXED

Whenever possible, the tool recommends the usage of external visibility instead of public, as the latter implies copying the arguments in the stack consuming more gas. The only file affected by this error is the Migrations.sol, which is not ours.

### Use of assembly: NOT FIXED
The tool, whenever assembly code is detected, triggers an alarm. These will not be fixed as the assembly code is necessary.

### Implicit visibility level: FIXED
This implies some of the state variables do not have visibility assigned, which by default assigns the public visibility.

### CODE REVIEW

File: benchmark/VRFGasHelper.sol [hunk 1]
```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.5.0;
+pragma solidity >=0.5.3 <=0.6.4;
```
Comment (clbartoli) [inline benchmark/VRFGasHelper.sol]: Why is here the version from 0.5.3 to 0.6.4 and not fixed? I thought this option was used in libraries, but I'm actually not sure..

Comment (girazoki) [reply benchmark/VRFGasHelper.sol]: you are right, we can live it as 0.6.4

File: package.json [hunk 2]
```diff
@@ -34,21 +34,21 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "elliptic-curve-solidity": "^0.2.1"
+    "elliptic-curve-solidity": "mariocao/elliptic-curve-solidity#chore/versions"
```
Comment (clbartoli) [inline package.json]: I guess you will changes this now that the PR has been merged :)

Comment (girazoki) [reply package.json]: indeed :)

**Pull request merged**